src/modules/scene/vscene.ts

```diff
@@ -209,94 +213,53 @@ export class VScene {
-            rotation: { x: 0, y: -0.5, z: 0 },
-            dimensions: { x: 1, y: 1, z: 1 },
-            color: { r: 1, g: 0.58, b: 0.86 }
-        });
-
-        await this.addEntity({
-            name: "cylinder",
-            type: "Shape",
-            shape: "cylinder",
-            position: { x: 1, y: 0, z: 0 },
-            rotation: { x: 0, y: -0.5, z: 0 },
-            dimensions: { x: 1, y: 1, z: 1 },
-            color: { r: 1, g: 0.58, b: 0.86 }
-        });
-
-        await this.addEntity({
-            name: "triangle",
-            type: "Shape",
-            shape: "triangle",
-            position: { x: 3, y: 0, z: 0 },
-            rotation: { x: 0, y: -0.5, z: 0 },
-            dimensions: { x: 1, y: 1, z: 1 },
-            color: { r: 1, g: 0.58, b: 0.86 }
-        });
-
-        const entityToDeleteID = uuidv4();
-
-        await this.addEntity({
-            name: "entityToDeleteByID",
-            id: entityToDeleteID,
-            type: "Shape",
-            shape: "triangle",
-            position: { x: 3, y: -2, z: 0 },
-            rotation: { x: 0, y: -0.5, z: 0 },
-            dimensions: { x: 1, y: 1, z: 1 },
-            color: { r: 1, g: 0.58, b: 0.86 }
-        });
-        this.deleteEntityById(entityToDeleteID);
-
-        await this.addEntity({
-            name: "entityToDeleteByName",
-            type: "Shape",
-            shape: "triangle",
-            position: { x: 3, y: 2, z: 0 },
-            rotation: { x: 0, y: -0.5, z: 0 },
-            dimensions: { x: 1, y: 1, z: 1 },
-            color: { r: 1, g: 0.58, b: 0.86 }
-        });
-        this.deleteEntityByName("entityToDeleteByName");
-
-        await this.addEntity({
-            name: "fox",
-            type: "Model",
-            modelUrl: "https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Fox/glTF/Fox.gltf",
-            position: { x: 5, y: 0, z: 0 },
-            rotation: { x: 0, y: -0.5, z: 0 },
-            dimensions: { x: 0.05, y: 0.05, z: 0.05 }
-        });
+        // aScene.createDefaultCameraOrLight(true, true, true);
+
+        const options = {
+            groundColor: BABYLON.Color3.White()
+        };
+
+        aScene.createDefaultEnvironment(options);
+
+        // This creates a light, aiming 0,1,0 - to the sky (non-mesh)
+        // const light = new BABYLON.HemisphericLight("light", new BABYLON.Vector3(0, 1, 0), aScene);
+
+        // Default intensity is 1. Let's dim the light a small amount
+        // light.intensity = 0.7;
+
+        // BABYLON.MeshBuilder.CreateGround("ground", { width: 30, height: 30 }, aScene);
+        const box = BABYLON.MeshBuilder.CreateBox("box1", {}, aScene);
+        box.position = new Vector3(5, 0.5, 5);
+
+
+        const avatarPos = new Vector3(0, 0, 0);
+
+        // Creates, angles, distances and targets the camera
+        const camera = new BABYLON.ArcRotateCamera(
+        // eslint-disable-next-line @typescript-eslint/no-magic-numbers
```

Comment:
Not needed because of global disable above.

Reply:
Fixed.